### PR TITLE
Fix typo in 'unique'

### PR DESCRIPTION
**Pull request by ellismg**

### CODE REVIEW

**Comment by ellismg** [comment]
Noticed this while doing some exploration and figured it was easy enough for me to come back here and fix it.

File: cli/azd/docs/md/azd.md [hunk 1]
```diff
@@ -0,0 +1,847 @@
+---
+title: Azure Developer CLI Preview reference
```
Comment (weikanglim) [inline cli/azd/docs/md/azd.md:2]: Just checking -- are you intending to include adding `azd.md` in this change as well?

Comment (ellismg) [reply cli/azd/docs/md/azd.md:2]: I did not!  Good catch.

**Review by vhvb1989 — APPROVED** [review]

**Review by wbreza — APPROVED** [review]
Funny how the spell linter didn't catch this previously.

**Pull request merged**